README.md

```diff
@@ -1,14 +1,36 @@
 ## [serotracker.com](https://serotracker.com/)
 
-### Github Pages Deployment
-Deployment of static assets for the application occurs on each push to master. The full configuration can be viewed [here](.github/workflows/deploy-gh-pages.yml). 
+### Github Pages/Heroku Deployment
+Deployment of static assets for the application occurs as follows:
+- Each push to `master` triggers a deployment to GitHub Pages. The GH action used is: `JamesIves/github-pages-deploy-action@releases/v3`
+- Each push to `staging-1`, `staging-2`, and `staging-3` triggers a deployment to Heroku. The GH action used is: `akhileshns/heroku-deploy@v3.0.4`
+
+The full configurations can be viewed [here](.github/workflows).   
+
+#### Adding Secrets
+Refer to [_creating encrypted secrets for a repository_](https://docs.github.com/en/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-a-repository).
+After adding secrets, they can be accessed in a deployment configuration in the following manner: `${{ secrets.<SECRET_NAME> }}`. 
+For example, see how the Mapbox API key is referenced [here](https://github.com/serotracker/sero-can-webapp/blob/master/.github/workflows/deploy-gh-pages.yml#L25).
```

Comment:
I see that this is how we configure secrets on the prod github pages deployment - what about the staging deployments??